content/help/faq/security/accessing-non-istio-services.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to disable mutual TLS on clients to access the Kubernetes API Server (or any services that don't have Istio sidecar)?
+weight: 60
+---
+
+When globally enable mutual TLS, the *global* destination rule matches all services in the cluster, including Kubernetes API Server. However, as the
```

Comment:
When mutual TLS is globally enabled, the *global* destinartion rule matches all services in the cluster, whether or not these services have an Istio sidecar. This includes the Kubernetes API server, as well as any non-Istio services in the cluster. To communicate with such services from services that have an Istio sidecar, you need to set a destination rule to exempt the service. For example:

Reply:
Done